query-node/schemas/forum.graphql

```diff
@@ -75,6 +76,7 @@ type ForumThread @entity {
   author: Membership!
 
   "Category the thread belongs to"
+  categoryId: ID!
```

Comment:
Is this field added for doing nested filtering, if yes please mark this field with the [comment](https://github.com/thesan/joystream/blob/cf06f473330110c157e5c18bc2886f3a00428b58/query-node/schemas/contentEvents.graphql#L24), it will help to remove all the redundant fields once nested filtering is available.

Reply:
Same thing here...

Reply:
Please see https://github.com/Joystream/joystream/pull/4317#discussion_r999076753